.travis.yml

```diff
@@ -1,14 +1,15 @@
 # Travis Continuous Integration Configuration File For ROS Control Projects
 # Author: Dave Coleman
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
This does not work. Travis does not support `xenial`, it will silently fall back to `precise`, which does not ship with docker (see https://github.com/ros-industrial/industrial_ci/pull/168 as well)